src/lib.ts

```diff
@@ -79,30 +81,29 @@ export default async function isMergable(opts: Options): Promise<Result> {
 	let pendingChecks: { name: string }[] = []
 	let missingChecks: { name: string }[] = []
 
-	let latestReviews: Map<
-		string,
-		Octokit.PullsListReviewsResponseItem
-	> = new Map()
+	let latestReviews: Map<string, typeof reviews.data[0]> = new Map()
```

Comment:
The changes to the definitions of these maps are probably the most odd feeling part of this change. I am actually surprised it worked.